src/kemal/base.cr

```diff
@@ -77,13 +77,6 @@ class Kemal::Base
   end
 
   private def prepare_for_server_start
```

Comment:
This is redundant now

Reply:
The method is empty in the base class, but necessary as it provides a way for subclasses like `Application` to hook in. Otherwise we'd have to reimplement `run` in `Application`.